.eleventy.js

```diff
@@ -19,6 +19,16 @@ module.exports = function(eleventyConfig) {
     return DateTime.fromJSDate(dateObj, {zone: 'utc'}).toFormat('yyyy-LL-dd');
   });
 
+  // Content for jsonfeed
+  eleventyConfig.addFilter('jsonContent', content => {
+    if (content) {
+      content = content.replace(/(\r\n|\n|\r)/gm, "");
+      content = content.replace(/\t/g, "\\t");
+      content = content.replace(/"/g, '\\"');
```

Comment:
Confused by these lines, I don’t think we want to globally remove new lines like this do we? Would it make sense to `JSON.stringify` this instead?

Reply:
You are correct. I don't remember why I went with that over `JSON.stringify` but it could definitely be used that way. I just made a small change for that and the output is similar in the final `feed.json` other than there being a lot of `\t` and `\n`. It's still valid json so that shouldn't be an issue.